docs/build/sdks/python/quickstart/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Pieces Wrapper Python SDK Quick Start
+description: Quickly set up the Pieces Wrapper Python SDK
+---
+
+# Python QuickStart
+
+Follow these steps to use the Pieces Python SDK.
+
+## Download Pieces OS
+
+:::info
+
+You must either have Pieces OS installed on your local machine or have access to a remote instance of Pieces OS to use this SDK.
+
+:::
+
+Download [**Pieces OS**](/installation-getting-started/pieces-os) for your operating system:
+
+- [**macOS**](/installation-getting-started/macos)
+- [**Windows**](/installation-getting-started/windows)
+- [**Linux**](/installation-getting-started/linux)
+
+## Install Python SDK
+
+Use pip to install the Pieces Python SDK using the following command:
+
+```sh
+pip install pieces_os_client
+```
+
+## Initialize SDK
+
+After installing the SDK, you must initialize the SDK with your base URL. The base URL will depend on your setup.
+
+If you are using a local instance of Pieces OS:
+- On macOS/Windows, use http://localhost:1000
+- On Linux, use http://localhost:5323
+
+If you are using a remote instance of Pieces OS, use the URL you have set up for that.
+
+## Getting started
```

Comment:
We should mention about our Wrapper here. and how is it different than the base SDK.